docs/replication-manager/replication-manager-for-pxc.md

```diff
@@ -0,0 +1,280 @@
+
+## Configuration steps
+
+1. Bootstrap the cluster on DC1-1 node:
+
+    ```bash
+    [root@DC1-1 ~]# /etc/init.d/mysql stop 
+    [root@DC1-1 ~]# /etc/init.d/mysql bootstrap-pxc
+    ```
+
+2. On DC1-2 node, stop MySQL, delete the content of the datadir to force the SST, and start MySQL:
+
+    ```bash
+    [root@DC1-2 ~]# /etc/init.d/mysql stop
+    [root@DC1-2 ~]# rm -rf /var/lib/mysql/*
+    [root@DC1-2 ~]# /etc/init.d/mysql start
+    ```
+
+3. Once the SST of DC1-2 node is completed, proceed on DC1-3:
+
+    ```bash
+    [root@DC1-3 ~]# /etc/init.d/mysql stop
+    [root@DC1-3 ~]# rm -rf /var/lib/mysql/*
+    [root@DC1-3 ~]# /etc/init.d/mysql start
+    ```
+
+4. The DC1 cluster uses a single [GTID](../glossary.md#gtid) sequence. To make sure `GTID_PURGED` is the same, run the following commands on DC1-1, DC1-2, and DC1-3 nodes:
+
+    ```bash
+    mysql> flush logs;
+    mysql> purge master logs to 'mysql-bin.000003';
+    ```
+
+    `mysql-bin.000003` is the last file returned from the show master logs. 
+
+5. Adjust the aformentioned steps to set up the DC2 and DC3 clusters accordingly.
+    
+6. Add privileges for replication between all nodes only for DC1: 
+
+    ```bash
+    GRANT REPLICATION CLIENT, REPLICATION SLAVE ON *.* TO 'repl'@'%' identified by 'replpass';
+    ```
+
+7. Add the replication table. For examle, create a table on DC1-1: 
+    
+    ```text
+    create database if not exists percona;
+    use percona;
+    CREATE TABLE `replication` (
+      `host` varchar(40) NOT NULL,
+      `weight` int(11) NOT NULL DEFAULT 0,
+      `localIndex` int(11) DEFAULT NULL,
+      `isSlave` enum('No','Yes','Proposed','Failed') DEFAULT 'No',
+      `lastUpdate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+      `lastHeartbeat` timestamp NOT NULL DEFAULT '1971-01-01 00:00:00',
+      `connectionName` varchar(64) NOT NULL,
+      PRIMARY KEY (`connectionName`,`host`),
+      KEY `idx_host` (`host`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `cluster` (
+      `cluster` varchar(31) NOT NULL,
+      `masterCandidates` varchar(255) NOT NULL,
+      `replCreds` varchar(255) NOT NULL,
+      PRIMARY KEY (`cluster`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `link` (
+      `clusterSlave` varchar(31) NOT NULL,
+      `clusterMaster` varchar(31) NOT NULL,
+      PRIMARY KEY (`clusterSlave`,`clusterMaster`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    CREATE TABLE `weight` (
+     `cluster` varchar(31) NOT NULL,
+     `nodename` varchar(255) NOT NULL,
+     `weight` int NOT NULL DEFAULT 0, 
+     PRIMARY KEY (`cluster`,`nodename`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+    ```
+
+    The replication table is written to by the replication manager. You do not need to adjust the table.
```

Comment:
is written to by?
The replication manager writes to the replication table.

Reply:
Updated